test/uk/gov/hmrc/tradergoodsprofilesrouter/service/GetRecordsServiceSpec.scala

```diff
@@ -79,7 +79,7 @@ class GetRecordsServiceSpec
         when(getRecordsConnector.fetchRecord(any, any, any, any)(any))
           .thenReturn(Future.successful(Left(badRequestErrorResponse)))
 
-        val result = await(sut.fetchRecord(eoriNumber, recordId, true))
+        val result = await(sut.fetchRecord(eoriNumber, recordId, ""))
```

Comment:
I think here it would be good to pass a non empty string as url to the connector and verofy that the connector call the fetchRecord with that string